packages/orbit-components/src/Card/components/Header.tsx

```diff
@@ -68,7 +68,7 @@ const Header = ({
         color="secondary"
       />
     )}
-    {actions && (
+    {React.isValidElement(actions) && (
```

Comment:
I've added this `React.isValisElement` function. In case we have `actions` in `CardSelection` component, then rendering `actions` here is redundant. In case `actions` is empty, we don't want to render `<Stack>` element below at all.